src/docs/product/session-replay/performance-overhead.mdx

```diff
@@ -14,7 +14,7 @@ Session Replay works by observing and recording changes to your web application'
 
 Sentry's Session Replay SDK takes several measures to avoid negatively impacting the performance of the page on which it's running:
 
-- At the time of publishing this document, the Session Replay plugin is an additional ~36KB gzipped.
+- As of version [7.78.0](https://blog.sentry.io/sentry-bundle-size-how-we-reduced-replay-sdk-by-35/) the Session Replay plugin is an additional ~36KB gzipped.
```

Comment:
@mydea is this 36kb the correct number for v7.78.0 still?

I'm trying to make things more concrete for people instead of saying "time of publishing" which is so hard to look up... 👎

Reply:
yeah that should still be correct!